src/Hive.Tests/Endpoints/ModsController.cs

```diff
@@ -167,6 +167,60 @@ public async Task SpecificModPluginFilter()
             Assert.True(value?.ID == "SongCore");
         }
 
+        [Fact]
+        public async Task SpecificModSpecificVersionStandard()
+        {
+            var controller = CreateController("next(true)", defaultPlugins);
+            var res = await controller.GetSpecificModSpecificVersion("BSIPA", "0.6.9"); // We will look for BSIPA.
+
+            Assert.NotNull(res); // Result must not be null.
+            Assert.NotNull(res.Result);
+            Assert.IsType<OkObjectResult>(res.Result); // The above endpoint must succeed.
+            var result = res.Result as OkObjectResult;
+            Assert.NotNull(result);
+            var value = result!.Value as SerializedMod;
+            Assert.NotNull(value); // We must be given a serialized mod back.
+
+            // This mod must be BSIPA.
+            Assert.True(value?.ID == "BSIPA");
+            // But! There's two versions of BSIPA, so we need to check that this version is REALLY 0.6.9.
+            Assert.True(value?.Version == new Versioning.Version(0, 6, 9));
```

Comment:
Both of these should probably be Assert.Equal

Reply:
I've made the change to the new tests, however this can be applied throughout the test class (and likely beyond), so I also left a TODO note at the top of the class for a cleanup PR